### ISSUE TITLE
containers-rootlessport dumps core

### ISSUE THREAD

**Issue by marusak** [containers/libpod#6018]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**
In Cockpit tests we noticed that podman sometimes dumps core. I don't actually have reproducer (yet) as  it does not happen always, but around in 1/3 of runs so slow to debug.

**Describe the results you received:**
I see podman coredumping:
```
  Command Line: containers-rootlessport                                         
    Executable: /usr/bin/podman                                                 
 Control Group: /user.slice/user-1000.slice/user@1000.service/io.podman.service 
          Unit: user@1000.service                                               
     User Unit: io.podman.service                                               
         Slice: user-1000.slice                                                 
     Owner UID: 1000 (admin)                                                    
       Boot ID: 4c93653f502c43e8ac08d914ce6dfad3                                
    Machine ID: 5ea7bfb5b3fa4a668686703decf4954a                                
      Hostname: localhost.localdomain                                           
       Storage: /var/lib/systemd/coredump/core.exe.1000.4c93653f502c43e8ac08d914ce6dfad3.7436.1588078680000000000000.lz4
       Message: Process 7436 (exe) of user 1000 dumped core.                    
@@@@@@@@@@@@@@@@                                                                
                Stack trace of thread 7444:                                     
                #0  0x0000562fce482251 runtime.raise (podman)                   
                #1  0x0000562fce467c6b runtime.dieFromSignal (podman)           
                #2  0x0000562fce4681fd runtime.sigfwdgo (podman)                
                #3  0x0000562fce467340 runtime.sigtrampgo (podman)              
                #4  0x0000562fce482543 runtime.sigtramp (podman)                
                #5  0x00007f191f830b20 __restore_rt (libpthread.so.0)           
                #6  0x0000562fce482251 runtime.raise (podman)                   
                #7  0x0000562fce467c6b runtime.dieFromSignal (podman)           
                #8  0x0000562fce467e0a runtime.crash (podman)                   
                #9  0x0000562fce452496 runtime.fatalpanic (podman)              
                #10 0x0000562fce451e56 runtime.gopanic (podman)                 
                #11 0x0000562fce428741 runtime.chansend (podman)                
                #12 0x0000562fce428145 runtime.chansend1 (podman)               
                #13 0x0000562fcf3ea3af github.com/containers/libpod/pkg/rootlessport.parent.func1 (podman)
                #14 0x0000562fcf3e98b8 github.com/containers/libpod/pkg/rootlessport.parent (podman)
                #15 0x0000562fcf3ea228 github.com/containers/libpod/pkg/rootlessport.init.0.func1 (podman)
                #16 0x0000562fcf73cacb main.main (podman)                       
                #17 0x0000562fce453bc6 runtime.main (podman)                    
                #18 0x0000562fce480841 runtime.goexit (podman)                  
@@@@@@@@@@@@@@@@                                                                
                Stack trace of thread 7436:                                     
                #0  0x0000562fce4827a3 runtime.futex (podman)                   
                #1  0x0000562fce44f6a6 runtime.futexsleep (podman)              
                #2  0x0000562fce42e353 runtime.notesleep (podman)               
                #3  0x0000562fce45844c runtime.stoplockedm (podman)             
                #4  0x0000562fce459d99 runtime.schedule (podman)                
                #5  0x0000562fce459f13 runtime.park_m (podman)                  
                #6  0x0000562fce47e733 runtime.mcall (podman)                   
                #7  0x0000562fce47e658 runtime.rt0_go (podman)                  
                #8  0x0000562fcf78af30 n/a (podman) 
```

I am attaching the whole backtrace from gdb: [bt.txt](https://github.com/containers/libpod/files/4545900/bt.txt)

If this backtrace is useless and you cannot figure out what is actually happening, I'll spend more to try to find reproducer.

**Output of `podman version`:**

```
[root@localhost ~]# podman version
Version:            1.9.0
RemoteAPI Version:  1
Go Version:         go1.13.9
OS/Arch:            linux/amd64
```

**Comment by baude** [comment]
@giuseppe PTAL ...

**Comment by baude** [comment]
@marusak did you get a core file from this? any chance you saved it?

**Comment by marusak** [comment]
> did you get a core file from this? any chance you saved it?

Our bots save those as well, one can be found [here](https://logs.cockpit-project.org/logs/pull-370-20200428-120538-fb42efc7-fedora-31-bots-804/TestApplication-testRunImageUser-fedora-31-127.0.0.2-2201-FAIL.core/). I also have some local ones if these would end up being unusable.

**Comment by giuseppe** [comment]
I've opened a PR here: https://github.com/containers/libpod/pull/6036

**Comment by matzew** [comment]
How can I get this fix, in 1.9.2 ?

I do not seem to get an update on my Fedora 32 ... 


```
sudo dnf update podman --enablerepo updates-testing
```

gives me nothing to update :disappointed:

**Comment by rhatdan** [comment]
It must not have gotten out to the mirrors yet.
You can grab the package from:
https://koji.fedoraproject.org/koji/buildinfo?buildID=1506922
